src/frogger.cpp

```diff
@@ -8,43 +9,6 @@
 #include "generated/model_output.hpp"
 #include "r_utils.hpp"
 
-int transform_simulation_years(
-    const Rcpp::List demp,
-    Rcpp::Nullable<Rcpp::NumericVector> r_sim_years) {
-
-  Rcpp::NumericVector Sx = demp["Sx"];
-  Rcpp::Dimension d = Sx.attr("dim");
-  // Simulation initialises state from first years input data (index 0)
-  // then runs for each year simulating this years (i) data using previous years
-  // state (i - 1) and this years input data (i).
```

Comment:
Can we keep this note somewhere? I think this is v easy to forget but quite important for making sense of the looping